doc/source/_static/css/custom.css

```diff
@@ -65,8 +69,8 @@ h6 {
   background-color: transparent;
 }
 .bd-content:after {
-/*   Commenting the code below to make more modifications after 2.10 release */
-/*   background: linear-gradient(
+  /*   Commenting the code below to make more modifications after 2.10 release */
+  /*   background: linear-gradient(
```

Comment:
not sure where this comes from.. but have passed 2.10 now.

Reply:
Looks like these changes came from elsewhere; after rebasing they're gone.